pkg/config/config.go

```diff
@@ -89,7 +89,7 @@ func (c *Config) Save() errors.Error {
 }
 
 // Read - retrieve config from a file
-func (c *Config) Read() errors.Error {
+func (c *Config) Read() error {
```

Comment:
I'm not sure but maybe here you can return Config instead of use a pointer

Reply:
large change, out of scope